astropy/io/ascii/misc.py

```diff
@@ -7,7 +7,7 @@
 """
 
 
-import collections
+import collections.abc
```

Comment:
we can't have this for the 2.0.x backport

Reply:
Yes, the warning is filtered for lxml, so should be fine.